pkg/validation/component.go

```diff
@@ -148,7 +146,7 @@ func (c *component) UpdateUser(ctx context.Context, realmName string, userID str
 	}
 
 	if kcUpdate || shouldRevokeAccreditations {
-		err = c.updateUserKeycloak(ctx, realmName, userID, user, shouldRevokeAccreditations)
+		err = c.updateUserKeycloak(validationCtx, user, shouldRevokeAccreditations)
```

Comment:
It is strange to provide a validation context while we are updating keycloak user. Even if it is the same fields which are part of it I think it is a bit misleading when we read it.